docs/public/deploying-a-devfile-using-odo.adoc

```diff
@@ -91,42 +93,36 @@ In our example, we will be using `java-spring-boot` to deploy a sample https://s
 
 == Deploying a Java Spring Boot® component to an OpenShift cluster
 
-In this example we will be deploying an https://github.com/odo-devfiles/springboot-ex[example Spring Boot® component] that uses https://maven.apache.org/install.html[Maven] and Java 8 JDK.
+In this example we will be deploying a sample project associated with the java-spring-boot devfile that uses https://maven.apache.org/install.html[Maven] and Java 8 JDK.
 
-. Download the example Spring Boot® component
+. Create a directory to store the source code for the component:
 +
 [source,sh]
 ----
- $ git clone https://github.com/odo-devfiles/springboot-ex
+ $ mkdir <directory-name>
 ----
 
-. Change the current directory to the component directory:
+. Create a component configuration of Spring Boot component-type named myspring and download its sample project:
 +
 [source,sh]
 ----
- $ cd <directory-name>
-----
+   $ odo create java-spring-boot myspring --downloadSource
```

Comment:
Could we maybe show both options (with `--downloadSource` and without it)? 
The reason is that I think that in most cases people will want to start with the existing code base and this might make the impression that they have to always use the repo that is defined in devfile.

Reply:
I left it out on the nodejs sample (https://github.com/openshift/odo/blob/e6b69bc46888a417eb6ea0b912d6e4494683c7c0/docs/public/deploying-a-devfile-using-odo.adoc#deploying-a-nodejs-component-to-an-openshift-cluster) so that there'd be an example with and without the flag

Reply:
I thoroughly believe we should leave out `--downloadSource` from this example. The purpose of the document is to give a basic example of Devfile and I don't think we should include this in here / add parameters.

Could we instead including this as a NOTE or a different instruction in a more advance document?

Basically, we should use the 30second-5minute-20minute philosophy for docs:

* 30 seconds - Grabbing the users attention & getting them to install and enable experimental mode (as per installation.md doc)
* 5 minutes - This document, getting them to go through a basic application that brings them up to speed quickly.
* 20 minutes - More advanced features, editing the devfile, etc (a future advanced document).